### ISSUE TITLE
Add background, standard deviation and local excess power in SpectrumDiscriminator and VariableSpectrumDiscriminator to DiscriminatedPoints1DData and DiscriminatedPoints2DData

### ISSUE THREAD

**Issue by cclaessens** [project8/katydid#137]
- [x] Add mean, variance and summed power variables to KTDiscriminatedPoints1DData and KTDiscriminatedPoints2DData

- [x]  Set new values during initialization of KTDiscriminatedPointsNDData in KTSpectrumDiscriminator

- [x]  Set new values during initialization  of KTDiscriminatedPointsNDData in KTVariableSpectrumDiscriminator

- [x] In KTVariableSpectrumDiscriminator, normalize new values in all occurrences of `if normalize`

**Comment by cclaessens** [comment]
I added the mean, sigma and the summed power as variables in KTDiscriminatedPoints1DData and KTDiscriminatedPoints2DData.

I also added the setting of these variables in one method of the KTVariableSpectrumDiscriminator: DiscriminateSpectrum  -> starting line 554 in the new feature branch feature/new-track-properties.
The equivalent needs to happen for all the other methods that are sued by the various slots to threshold and collect points.

In `if normalize == True` I only changed the value, the mean and the variance. I did not change the threshold because I haven't made up my mind about leaving the threshold as it is or storing the equivalent in normalized power. We should probably do the later.
